flybywiresim/a32nx
# fix(ewd): Fix for wing anti-ice not appearing on EWD when using keyboard shortcut

Fixes #[issue_no]

## Summary of Changes

At present, the correct displaying behaviour for the EWD and SD bleed page with respect to wing anti-ice, only occurs if the wing anti-ice switch is pressed in sim. If the keyboard shortcut is used, this does not work as expected. This PR fixes this issue.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Spawn in flight
2. Turn on wing anti-ice via the switch and the keyboard shortcut 'H'
3. Confirm correct displaying behaviour on both the EWD and the bleed page.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
